test/policies/supported_permission_policy_scope_test.rb

```diff
@@ -23,6 +25,9 @@ class SupportedPermissionPolicyScopeTest < ActiveSupport::TestCase
 
     @app_four_pat_permission = create(:non_delegatable_supported_permission, application: @app_three, name: "pat")
     @app_four_sat_permission = create(:delegatable_supported_permission, application: @app_three, name: "sat")
+
+    @api_only_app_nat_permission = create(:non_delegatable_supported_permission, application: @api_only_app, name: "nat")
```

Comment:
These names 😄

Reply:
Yeah I just went with it 😅 this suite could definitely have a bit more of a refactor, but not right now!